src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/config/HmppsAllocationsExceptionHandler.kt

```diff
@@ -9,11 +9,11 @@ import org.springframework.http.HttpStatus.METHOD_NOT_ALLOWED
 import org.springframework.http.ResponseEntity
 import org.springframework.http.converter.HttpMessageConversionException
 import org.springframework.security.access.AccessDeniedException
-import org.springframework.web.HttpRequestMethodNotSupportedException
```

Comment:
this is a Servlet exception, no more servlets in reactive. the equivalent is `MethodNotAllowedException`